src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
According to the checklist, the `get()` method should return `Optional<MovieSession>` on the DAO layer, while on the service layer it should return `MovieSession` directly. Please make sure that the implementation of this method on the DAO layer returns `Optional<MovieSession>`.